### PR TITLE
Feature/form post rebase

### PR DESCRIPTION
**Pull request by lilianaweimer**
## Pull request checklist
Please check if your PR fulfills the following requirements:
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type
 
 
Please check the type of change your PR introduces:
- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Testing
- [ ] Other (please describe): 

## What is the current behavior?

- Posting to the database is working, but nothing is coming back

## What is the new behavior?

- No change, but different errors

## Does this introduce a breaking change?
- [ ] Yes
- [x] No

### CODE REVIEW

File: code/api/src/modules/user/query.js [hunk 1]
```diff
@@ -12,10 +12,10 @@ export const users = {
 }
 
 //Update
-export const userUpdate = {
-  type: new GraphQLList(UserType),
-  resolve: update
-}
+// export const userUpdate = {
+//   type: new GraphQLList(UserType),
```
Comment (cgaddis36) [inline code/api/src/modules/user/query.js:16]: will remove these comments on next pr

File: code/api/src/modules/user/resolvers.js [hunk 2]
```diff
@@ -28,39 +28,39 @@ export async function create(parentValue, { name, email, password }) {
...
+    { where: { id: auth }  }
+    )
+    } else {
+      throw new Error('Email already exists')
+    }
+  } else {
+    return await models.User.update({
+      name,
+      email,
+      description,
```
Comment (cgaddis36) [inline code/api/src/modules/user/resolvers.js:54]: lets see if the indentation stays this time...

File: code/api/src/modules/user/types.js [hunk 3]
```diff
@@ -13,7 +13,12 @@ const UserType = new GraphQLObjectType({
     password: { type: GraphQLString },
     role: { type: GraphQLString },
     createdAt: { type: GraphQLString },
-    updatedAt: { type: GraphQLString }
+    updatedAt: { type: GraphQLString },
+    city: { type: GraphQLString },
+    state: { type: GraphQLString },
+    zip: { type: GraphQLString },
+    shipping_address: { type: GraphQLString },
+    description: { type: GraphQLString }
```
Comment (cgaddis36) [inline code/api/src/modules/user/types.js:21]: looks good here

File: code/web/src/modules/user/ProfileForm.js [hunk 4]
```diff
@@ -19,11 +20,13 @@ class ProfileForm extends PureComponent {
     super(props)
 
     this.state = {
+      name: props.user.details.name,
       email: props.user.details.email,
-      address: props.user.details.shippingAddress,
+      shipping_address: props.user.details.shipping_address,
```
Comment (jdvolk) [inline code/web/src/modules/user/ProfileForm.js:25]: thanks for fixing the shipping address

File: code/web/src/modules/user/api/actions.js [hunk 5]
```diff
@@ -66,6 +67,44 @@ export function login(userCredentials, isLoading = true) {
   }
 }
 
+export function updateUser(userDetails) {
+  return dispatch => {
+    return axios.post(routeApi, mutation({
+      operation: 'userUpdate',
+      variables: userDetails,
```
Comment (jdvolk) [inline code/web/src/modules/user/api/actions.js:74]: looks good here

**Pull request merged**